server/sonar-server/src/main/java/org/sonar/server/qualitygate/ws/GetByProjectAction.java

```diff
@@ -95,8 +95,8 @@ public void handle(Request request, Response response) throws Exception {
       // As ComponentFinder doesn't handle organization yet, we only check here that the project belongs to the organization
       wsSupport.checkProjectBelongsToOrganization(organization, project);
 
-      if (!userSession.hasComponentPermission(USER, project) &&
-        !userSession.hasComponentPermission(ADMIN, project)) {
+      if (userSession.hasComponentPermission(USER, project) ||
+        userSession.hasComponentPermission(ADMIN, project)) {
         throw insufficientPrivilegesException();
```

Comment:
Something is strange : we should allow user with browse or admin permission to access to this WS => code should be 
{code}
   if (!userSession.hasComponentPermission(USER, project) || !userSession.hasComponentPermission(ADMIN, project)) {
{code}